Formula/o/orbuculum.rb

```diff
@@ -5,6 +5,11 @@ class Orbuculum < Formula
   sha256 "ccdd86130094001a0ab61e5501a6636e12c82b0b44690795a2911c65c5618c46"
   license "BSD-3-Clause"
 
+  livecheck do
+    url :stable
+    regex(/^V?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

The Git tags use a capital V (e.g., `V2.1.0`) but the standard regex for Git tags like `1.2.3`/`v1.2.3` is case insensitive, so we should simply use that for the sake of standardization.